servicetalk-examples/grpc/compression/src/main/java/io/servicetalk/examples/grpc/compression/CompressionExampleServer.java

```diff
@@ -68,5 +65,10 @@ public static void main(String... args) throws Exception {
         public Single<HelloReply> sayHello(final GrpcServiceContext ctx, final HelloRequest request) {
             return succeeded(HelloReply.newBuilder().setMessage("Hello " + request.getName()).build());
         }
+
+        @Override
+        public ServiceFactory bindService() {
+            return new ServiceFactory(this, SUPPORTED_ENCODINGS);
```

Comment:
consider adding a comment clarifying the configuration can be done here in the `bindService` method or explicitly on the `listenAndAwait` method above.

Reply:
done.